src/main/java/no/nav/veilarbperson/client/pdl/PdlClientImpl.java

```diff
@@ -64,44 +66,95 @@ public PdlClientImpl(String pdlUrl, Supplier<String> userTokenProvider, Supplier
         this.hentTilrettelagtKommunikasjonQuery = FileUtils.getResourceFileAsString("graphql/hentTilrettelagtKommunikasjon.gql");
     }
 
+    @Deprecated
     @Override
     public HentPerson.Person hentPerson(Fnr personIdent) {
+        secureLog.info("Kaller deprecated funksjon hentperson");
```

Comment:
Logger er lagt på for å forsikre meg om at disse funksjonene ikke blir brukt lenger. Funskjonene og logglinjene skal fjernes før merge.